modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -12,6 +12,43 @@
 	}
 }
 
+#portlet_com_liferay_headless_builder_web_internal_portlet_HeadlessBuilderPortlet {
+	.card-type-asset {
+		.card-body {
+			padding-bottom: 1.5rem;
```

Comment:
@cgmonte Since all paddings have the same value, you can replace those 4 lines with `padding: 1.5rem`.

Reply:
of course! :facepalm: :laughing: